ministryofjustice/correspondence_tool_staff
# CT-2926 no case edit history if no changes

## Description

Solve the issue for adding new entry to case history even there is nothing being changed
## Self-review checklist


### Screenshots

before any change 
<img width="953" alt="Screenshot 2020-07-16 at 13 40 23" src="https://user-images.githubusercontent.com/26111671/87671988-233ecb00-c76a-11ea-8714-144cdfcc4077.png">

Choose a section and click 'change', and then click "continue" without changing anything
<img width="835" alt="Screenshot 2020-07-16 at 13 40 38" src="https://user-images.githubusercontent.com/26111671/87672008-2f2a8d00-c76a-11ea-99da-3b19f1ef292d.png">
Choose same section, then change something and click "continue" 
<img width="870" alt="Screenshot 2020-07-16 at 13 40 52" src="https://user-images.githubusercontent.com/26111671/87672134-5c773b00-c76a-11ea-9534-61d042524076.png">
<img width="1010" alt="Screenshot 2020-07-16 at 13 40 56" src="https://user-images.githubusercontent.com/26111671/87672144-5ed99500-c76a-11ea-95ce-3b8727fa6cec.png">


### Related JIRA tickets

https://dsdmoj.atlassian.net/browse/CT-2926
### Deployment

This PR is to fix the issue of adding new entry into case history even you didn't change anything
### Manual testing instructions

login as branson office
choose a offender case 
click a section from case detail part e.g. subject detail
Click the 'continue' button without changing anything 
you should see 'No change were made' and no adding record in case history
Click the same section again and change some detail 
you should see 'case updated' and a new entry in the case history
